contracts/orderbook/Storage/OrderBookEvents.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.8.0;
+import "../IOrderBook.sol";
+import "../../governance/PausableGuardian_0_8.sol";
+import "@openzeppelin-4.3.2/utils/structs/EnumerableSet.sol";
+
+contract OrderBookEvents is PausableGuardian_0_8 {
```

Comment:
its either events or state. extract events to separate file

Reply:
there was some mixing due to imports but I can separate them.